.github/workflows/deploy-docs.yml

```
@@ -0,0 +1,126 @@
+        uses: actions/configure-pages@v4
+
+      - name: Upload artifact
+        uses: actions/upload-pages-artifact@v3
+        with:
+          path: ./build
+
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)